src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -0,0 +1,103 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+
+import {
+  mockUser,
+  mockItf,
+  mockDocumentUpload,
+  mockPaymentInformation,
+  mockApplicationSubmit,
+} from './all-claims.cypress.helpers.js';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['maximal-test', 'minimal-test', 'newOnly-test'],
+    fixtures: {
+      data: path.join(__dirname, 'data'),
+    },
+    pageHooks: {
+      introduction: () => {
+        // Hit the start button
+        cy.findAllByText(/start/i, { selector: 'button' })
+          .first()
+          .click();
+
+        // Click past the ITF message
+        cy.findByText(/continue/i, { selector: 'button' }).click();
+      },
+      'disabilities/rated-disabilities': () => {
+        cy.get('@testData').then(data => {
+          data.ratedDisabilities.forEach((disability, index) => {
+            if (disability['view:selected']) {
+              cy.get(`input[name="root_ratedDisabilities_${index}"]`).click();
+            }
+          });
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+      'payment-information': () => {
+        cy.get('@testData').then(data => {
+          if (data['view:bankAccount']) {
+            cy.get('form.rjsf').then($form => {
+              const editButton = $form.find('.usa-button-primary.edit-button');
+              if (editButton) editButton.click();
+            });
+
+            cy.fillPage();
+            cy.findByText(/save/i, { selector: 'button' }).click();
+          }
+
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+    },
+    setupPerTest: () => {
+      // Set up signed in session.
+      window.localStorage.setItem('hasSession', true);
```

Comment:
Do you think it would be useful to have a setting for defining the `localStorage` items? For example, rather than manually adding this to the `setupPerTest function`:
```javascript
window.localStorage.setItem('hasSession', true);
window.localStorage.setItem('DISMISSED_ANNOUNCEMENTS', '*');
```
We could have a setting called `setLocalStorage` that is optionally passed into the config. Maybe something like:
```javascript
{ ...testConfig,
   setLocalStorage({
   'hasSession': true,
   'DISMISSED_ANNOUNCEMENTS': '*'
})
}
```

Reply:
Setting `localStorage` items generally isn't form-specific, so I'm not sure the test config is the place for it. There aren't that many items to track in there, and the items being set will probably end up being handled in a related helper or custom command, like `hasSession` being pulled out into a `login` command. I could see the `DISMISSED_ANNOUNCEMENTS` ending up in some `dismissAnnouncements` command or something as well.

Reply:
That makes sense. I was thinking there were more `localStorage` items set for some reason, but after doing a search in the repo, `DISMISSED_ANNOUNCEMENTS` and `hasSession` are literally the only items that are ever set. Which in that case, it does makes sense to have them manually handled or set in a custom command like you mentioned.